stl/inc/chrono

```diff
@@ -617,6 +618,100 @@ namespace chrono {
     };
 
     using high_resolution_clock = steady_clock;
+
+#ifdef _HAS_CXX20
+    // [time.duration.io]
+    class _Units_suffix_helper {
+    public:
+        template <class _CharT, class _Period>
+        _NODISCARD static constexpr const _CharT* _Get_literal() {
+            for (const auto& _Entry : _Suffixes) {
+                if (_Entry._Num == _Period::num && _Entry._Den == _Period::den) {
+                    return _STD get<const _CharT*>(_Entry._Name);
+                }
+            }
+            return nullptr;
+        }
+
+        template <class _CharT>
+        _NODISCARD static _CharT* _Get_general(_CharT* _Rnext, intmax_t _Num, intmax_t _Den) {
+            // Returns the head pointer of the string, built in reverse.
+            _STL_INTERNAL_CHECK(_Num > 0 && _Den > 0);
+            *--_Rnext = '\0';
+            *--_Rnext = 's';
+            *--_Rnext = ']';
+            if (_Den != 1) {
+                _Rnext    = _UIntegral_to_buff(_Rnext, static_cast<uintmax_t>(_Den));
+                *--_Rnext = '/';
+            }
+            _Rnext    = _UIntegral_to_buff(_Rnext, static_cast<uintmax_t>(_Num));
+            *--_Rnext = '[';
+            return _Rnext;
+        }
+
+    private:
+        struct _Suf_names {
+            const intmax_t _Num;
+            const intmax_t _Den;
+            const pair<const char*, const wchar_t*> _Name;
+        };
+
+#define _LITERAL_SUFFIX(_TYPE, _SUFFIX) \
+    { _TYPE::num, _TYPE::den, _STD make_pair(_SUFFIX, L##_SUFFIX) }
+#define _LITERAL_SUFFIX2(_TYPE, _NARROW_SUFFIX, _WIDE_SUFFIX) \
+    { _TYPE::num, _TYPE::den, _STD make_pair(_NARROW_SUFFIX, _WIDE_SUFFIX) }
+
+        static constexpr _Suf_names _Suffixes[]{
+            // clang-format off
+            _LITERAL_SUFFIX(atto, "as"),
+            _LITERAL_SUFFIX(femto, "fs"),
+            _LITERAL_SUFFIX(pico, "ps"),
+            _LITERAL_SUFFIX(nano, "ns"),
+            _LITERAL_SUFFIX2(micro, "us", L"\u00b5s"),

```

Comment:
After chatting with @StephanTLavavej, I think it's best that we stick with a consistent "us" suffix for micro, for both `char` and `wchar_t`.  He pointed out that, as far as we know, there is nothing else in the CRT/STL that has different behavior based on the character type so it would be a little strange for this to be the only thing behaving in this way, and using `wprintf` without the Unicode output hack (more info [here](https://docs.microsoft.com/en-us/cpp/c-runtime-library/reference/setmode?view=vs-2019#example-use-_setmode-to-change-stdout)) breaks with "µs". 

![microseconds](https://user-images.githubusercontent.com/46756417/96653269-89009d00-12ed-11eb-8280-9659c5df2aac.png)

This in conjunction with the fact that there isn't much to be gained by using "µs", I think we should just err on the side of just "us" for micro 😸